### PR TITLE
#2269575: Implement "Path has url alias" condition

### PR DESCRIPTION
**Pull request by Nebel54**

### CODE REVIEW

File: src/Plugin/Condition/UrlAlias.php [hunk 1]
```diff
@@ -0,0 +1,86 @@
...
+
+/**
+ * Provides a 'Path has URL alias' condition.
+ *
+ * @Condition(
+ *   id = "rules_path_has_url_alias",
+ *   label = @Translation("Path has URL alias"),
+ *   context = {
+ *     "path" = {
+ *       "label" = "The path",
```
Comment (fago) [inline src/Plugin/Condition/UrlAlias.php:24]: Is that the existing d7 label?. Also should use @ Translation.

File: src/Plugin/Condition/UrlAlias.php [hunk 2]
```diff
@@ -0,0 +1,86 @@
...
+ *    }
+ *   }
+ * )
+ *
+ * @todo: Add access callback information from Drupal 7.
+ * @todo: Add group information from Drupal 7.
+ */
+class UrlAlias extends ConditionPluginBase implements ContainerFactoryPluginInterface {
+
+  protected $aliasManager;
```
Comment (fago) [inline src/Plugin/Condition/UrlAlias.php:41]: Needs docs.

File: src/Tests/Condition/UrlAliasConditionTest.php [hunk 3]
```diff
@@ -0,0 +1,68 @@
...
+/**
+ * Tests the node conditions.
+ */
+class UrlAliasConditionTest extends PathUnitTestBase {
+
+  /**
+   * {@inheritdoc}
+   */
+  public static $modules = array('system', 'rules', 'path');
+  protected $alias_manager;
```
Comment (fago) [inline src/Tests/Condition/UrlAliasConditionTest.php:25]: Needs new lines and docs.

File: src/Tests/Condition/UrlAliasConditionTest.php [hunk 4]
```diff
@@ -0,0 +1,68 @@
...
+namespace Drupal\rules\Tests\Condition;
+
+use Drupal\system\Tests\Entity\EntityUnitTestBase;
+use Drupal\Core\Path\AliasStorage;
+use Drupal\Core\Database\Database;
+use Drupal\Core\Path\AliasManager;
+use Drupal\system\Tests\Path\PathUnitTestBase;
+
+/**
+ * Tests the node conditions.
```
Comment (fago) [inline src/Tests/Condition/UrlAliasConditionTest.php:17]: outdated

**Comment by paranojik** [comment]
Trying to push this forward in https://github.com/fago/rules/pull/31.

**Pull request closed without merging**